std_msgs/Header header
float64 yaw_rate_float_ 
float64 lateral_acceleration_float_
float64 longitudinal_acceleration_float_
float64 wheel_speed_float_
float64 velocity
bool velocity_data_active
float64 velocity_x
float64 velocity_y
float64 velocity_z
float64 steering_angle_float_
float64 global_dist_mts_float_
